archr/targets/flight.py

```diff
@@ -27,13 +34,33 @@ def get_channel(self, channel_name):
         return channel
 
     def open_channel(self, channel_name):
+        def patch_log(nc):
+            def print_verbose(self, s):
+                assert isinstance(s, str), "s should be str"
+                l.debug(s)
+            nc.verbose = True
+            nc._print_verbose = print_verbose.__get__(nc, type(nc))
```

Comment:
hello?? hello???? what the everloving fuck is this?

Reply:
This is why we use python, right?